src/modules/core/components/Preloaders/MiniSpinnerLoader.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+
+import Heading, { Appearance as HeadingAppearance } from '~core/Heading';
+import { SimpleMessageValues } from '~types/index';
+
+import SpinnerLoader, { Appearance } from './SpinnerLoader';
+import styles from './MiniSpinnerLoader.css';
+
+interface Props {
+  appearance?: Appearance;
+  titleAppearance?: HeadingAppearance;
+  title?: string | MessageDescriptor;
+  /** Values for text (react-intl interpolation) */
+  titleTextValues?: SimpleMessageValues;
+  loadingText?: string | MessageDescriptor;
+  className?: string;
+  loadingTextClassName?: string;
+}
+
+const MiniSpinnerLoader = ({
+  appearance,
+  titleAppearance = { size: 'normal', weight: 'bold' },
+  title,
+  titleTextValues,
+  loadingText,
+  className,
+  loadingTextClassName,
+}: Props) => {
+  return (
+    <div className={className}>
+      {title && (
+        <Heading
+          text={title}
+          textValues={titleTextValues}
+          appearance={titleAppearance}
+        />
+      )}
+      <SpinnerLoader appearance={appearance} />
+      {loadingText && (
+        <span className={loadingTextClassName || styles.loadingText}>
```

Comment:
I don't think we should have this customizable, but it your choice in the end, so I'm fine if you decide to leave it in.

Reply:
In some components the styles are a bit different - like when we have it in a dialog and a couple of other instances. So I prefer to leave it like that.